cmd/kops/create_secret_dockerconfig.go

```diff
@@ -35,7 +35,12 @@ var (
 	createSecretDockerconfigLong = templates.LongDesc(i18n.T(`
 	Create a new docker config, and store it in the state store.
 	Used to configure docker on each master or node (i.e. for auth)
-	Use update to modify it, this command will only create a new entry.`))
+	Use update to modify it, this command will only create a new entry.
+	
+	After creating a dockerconfig secret, a .docker/config.json file will 
+	be added to all newly created nodes without running kops update.
+	This file should also apply auth to containerd, which is the default
+	container runtime in kops 1.20 and above.`))
```

Comment:
```suggestion
	After creating a dockerconfig secret, a /root/.docker/config.json file will 
	be added to newly created nodes (without running kops update).
	This should also work when using containerd, which is the default
	container runtime in kOps 1.20 and above.`))
```

Reply:
Can we say with confidence that it _will_ work? not only _should_ :)

Reply:
Sure! 🤣